package/kernel/linux/modules/crypto.mk

```diff
@@ -437,6 +437,38 @@ endef
 $(eval $(call KernelPackage,crypto-hw-padlock))
 
 
+define KernelPackage/crypto-hw-rockchip
```

Comment:
since it's rockchip only, would it make sense to move this into `target/linux/rockchip/modules.mk`?

Reply:
Yes, it makes sense move the module to the rockchip target, done.